sdk/core/src/main/java/org/apache/mesos/scheduler/DefaultScheduler.java

```diff
@@ -157,12 +177,11 @@ private void initialize(SchedulerDriver driver) throws InterruptedException {
                 deployPlanManager);
         planCoordinator = new DefaultPlanCoordinator(planManagers, planScheduler);
         planCoordinator.subscribe(this);
-        logger.info("Done initializing.");
+        LOGGER.info("Done initializing.");
     }
 
     private void initializeGlobals(SchedulerDriver driver) {
-        logger.info("Initializing globals...");
-        stateStore = new CuratorStateStore(frameworkName, zkConnectionString);
```

Comment:
What about the additional CuratorStateStore instance being created and passed to DefaultStepFactory in a create() method? (See L72 in original DefaultScheduler.java)